src/local_scheduler/test/local_scheduler_tests.cc

```diff
@@ -83,31 +88,31 @@ LocalSchedulerMock *LocalSchedulerMock_init(int num_workers,
   memset(mock, 0, sizeof(LocalSchedulerMock));
   mock->loop = event_loop_create();
   /* Bind to the local scheduler port and initialize the local scheduler. */
-  UT_string *plasma_manager_socket_name = bind_ipc_sock_retry(
+  std::string plasma_manager_socket_name = bind_ipc_sock_retry(
       plasma_manager_socket_name_format, &mock->plasma_manager_fd);
   mock->plasma_store_fd =
       connect_ipc_sock_retry(plasma_store_socket_name, 5, 100);
-  UT_string *local_scheduler_socket_name = bind_ipc_sock_retry(
+  std::string local_scheduler_socket_name = bind_ipc_sock_retry(
       local_scheduler_socket_name_format, &mock->local_scheduler_fd);
   CHECK(mock->plasma_store_fd >= 0 && mock->local_scheduler_fd >= 0);
 
-  UT_string *worker_command;
-  utstring_new(worker_command);
-  utstring_printf(worker_command,
-                  "python ../../../python/ray/workers/default_worker.py "
-                  "--node-ip-address=%s --object-store-name=%s "
-                  "--object-store-manager-name=%s --local-scheduler-name=%s "
-                  "--redis-address=%s:%d",
-                  node_ip_address, plasma_store_socket_name,
-                  utstring_body(plasma_manager_socket_name),
-                  utstring_body(local_scheduler_socket_name), redis_addr,
-                  redis_port);
+  size_t worker_command_size =
```

Comment:
For this part of the code let's use stringstreams from the c++ standard library to construct the worker command. It is cleaner and easier to understand.

Reply:
Should be fixed in latest commit!